src/index.html

```diff
@@ -1,14 +1,28 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <section class="stop-watch">
+      <div class="stop-watch__time stop-watch__time--0">0</div>
+
+      <div class="stop-watch__time stop-watch__time--15">15</div>
+
+      <div class="stop-watch__time stop-watch__time--30">30</div>
+
+      <div class="stop-watch__time stop-watch__time--45">45</div>
+
+      <div class="stop-watch__seconds"></div>
+
+      <div class="stop-watch__minutes"></div>
+
+      <div class="stop-watch__center"></div>
```

Comment:
i think it's to many blank lines. Also you can youse mix classes for watch-hands like this :
```suggestion
      <div class="stop-watch__hand stop-watch__hand--sec"></div>
      <div class="stop-watch__hand stop-watch__hand--min"></div>
```